src/main/kotlin/minesweeper/domain/Cells.kt

```diff
@@ -0,0 +1,12 @@
+package minesweeper.domain
+
+class Cells(val cells: List<Cell>) {
+
+    companion object {
+        fun of(positions: List<Position>, minePositions: List<Position>): Cells {
+            return positions.map { position ->
+                Cell.of(position, minePositions.contains(position))
```

Comment:
```suggestion
                Cell.of(position, minePositions.cellType(position))
```
`List<Position>` 을 일급 컬렉션으로 바꾸고 메소드를 추가해 보면 어떨까요?